exercise_chapter11/inverse_dict.py

```diff
@@ -0,0 +1,27 @@
+"""
+This module contains a code for exercises 11-2 related to:
+Think Python, 2nd Edition
+Chapter 11: Dictionaries
+
+This function reverses a dictionary and uses the dictionary's 
+setdefault() method to generate key-value pairs.
+"""
+
+def invert_dict(dict_in):
+    """
+    Reverse the dictionaries, using the method setdefault()
+
+    dict_in: dict
+
+    return: reverse dict 
+    """
+    
+    inverse = {}
+    
+    for key in dict_in:
+        value = dict_in[key]
+        inverse.setdefault(value, []).append(key)
+    
+    return inverse
+
+invert_dict(dict_in=())
```

Comment:
According to the docstring, the parameter to be passed has a data type of dict, as I see you are passing an empty tuple.